### ISSUE TITLE
Not possible to pipe logs for connectors to file

### ISSUE THREAD

**Issue by artem-shelkovnikov** [elastic/connectors-python#1460]
## Bug Description

It's not possible to pipe logs for running connectors to a file.

### To Reproduce
1. Run `make run >> logs.txt` or `bin/elastic-ingest >> logs.txt` with installed service
2. Observe the content of `logs.txt`
3. File is empty

## Expected behavior
All server logs are piped to the file

**Comment by seanstory** [comment]
I think this is a STDERR thing. My bash is rusty, but I feel like you need a `2>` to get STDERR. This works:

```
Seans-MacBook-Pro:connectors-python seanstory$ bin/elastic-ingest 2> output.txt
Seans-MacBook-Pro:connectors-python seanstory$ cat output.txt 
[FMWK][10:27:48][INFO] Running connector service version 8.11.0.0
[FMWK][10:27:48][INFO] Loading config from /Users/seanstory/Desktop/Dev/connectors-python/connectors/../config.yml
[FMWK][10:27:48][INFO] Preflight checks...
...
[FMWK][10:27:48][INFO] Bye
```

**Comment by seanstory** [comment]
also, `nohup bin/elastic-ingest` works as expected.

**Comment by artem-shelkovnikov** [comment]
IMO this should be default behaviour though - normal logs to stdout, error logs to stderr.

Curious - why do we have this behaviour override?

**Comment by artem-shelkovnikov** [comment]
Yeah it's an easy change, but IMO really important for predictability of service (and I often needed to pipe logs to a file, so I expect there will be other people with same expectations)
